samples/storage_csm_powerflex_v220.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: storage.dell.com/v1alpha1
```

Comment:
should we update this version if v1alpha1 is removed now?

Reply:
i will rebase my branch from main 
Probably that will change the version